spec/support/shared_examples_for_geckoboard_reports.rb

```diff
@@ -109,13 +94,13 @@ def mock_expectations(exit_without_ping = nil)
 RSpec.shared_examples 'returns valid items structure' do
```

Comment:
I think you also need to define `subject(:report) { described_class.new }` here. At the moment tests that use this shared example are failing with the error `undefined local variable or method 'report'`, I think because the `report` defined on line 8 isn't in scope here.

Reply:
Ah, I forgot that there were multiple groups of shared example in that file and just did a search/replace on the whole file. I'll change it back in those three places.